README.md

```diff
@@ -141,9 +144,7 @@ If you like this project, you are most welcome to Contribute.
 
```

Comment:
- [ ] It would be nice if you rename your branch name to something like `feature/finalize-capstone` to make your pull request more professional 👍